scripts/apps/relations/directives/RelatedItemsDirective.ts

```diff
@@ -217,15 +219,23 @@ export function RelatedItemsDirective(
             /**
             * Get related items for fieldId
             */
-            scope.refreshRelatedItems = () => {
+            scope.loadRelatedItems = () => {
                 scope.loading = true;
                 relationsService.getRelatedItemsForField(scope.item, scope.field._id)
                     .then((items) => {
                         scope.relatedItems = items;
                         scope.loading = false;
                     });
             };
-            scope.refreshRelatedItems();
+            scope.loadRelatedItems();
+
+            scope.getThumbnailForItem = (item) => {
+                if (!item.associations?.featuremedia) {
+                    return null;
+                }
+
+                return getViewImage(item.associations?.featuremedia as IArticle);
```

Comment:
Avoid using type casting via `as` where possible. In this case, you can do a check similar to https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/apps/workspace/content/constants.ts#L131